README.md

```diff
@@ -42,3 +55,55 @@ To run the stack in production, you can run the following
 ```
 
 This will start the stack in production mode.
+
+## Docker Commands
+
+Here are some useful Docker commands for managing the stack:
+
+1. Build Docker images:
+
+   ```bash
+   docker-compose build
+   ```
+
+2. Start services:
+  
+   ```bash
+   docker-compose up
+   ```
+
+   For detached mode:
+
+   ```bash
+   docker-compose up -d
+   ``` 
+
+3. Stop services:
+
+   ```bash
+   docker-compose down
+   ```
+
+4. Rebuild a specific service (e.g., backend):
+
+   ```bash
+   docker-compose build backend
+   ```
+
+5. Restart a specific service:
+
+   ```bash
+   docker-compose restart backend
+   ```
+
+6. View service status:
+
+   ```bash
+   docker-compose ps
+   ```
+
+7. View logs:
+   - All services: `docker-compose logs`    
+   - Specific service: `docker-compose logs backend`
+
+Remember to run these commands from the directory containing your `docker-compose.yml` file.
```

Comment:
The stack uses a helper script, `run_stack.sh`, to select the correct `docker-compose.*.yml` files for a given environment; we're not using `docker compose` commands directly in this project for that reason.

I'd also rather not clutter the README with general information about Docker and Compose.